packages/lab/src/tabs/Tab.css

```diff
@@ -1,37 +1,38 @@
-/* class applied to root Tab element */
+/* Class applied to root Tab element */
 .uitkTab {
-  --uitkFocusVisible-inset: 2px 2px 4px 2px;
   --uitkEditableLabel-padding: 0;
```

Comment:
the custom value for focusVisible inset was needed, focusvisible is now broken on Tabs